airflow/providers/google/cloud/operators/dataproc.py

```diff
@@ -15,7 +15,16 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-"""This module contains Google Dataproc operators."""
+"""
+This module contains Google Dataproc operators.
+
+.. spelling::
+
+    preemptible
+    Preemptibility
+    preemptibility
+    InstanceGroupConfig
```

Comment:
shouldn't these be in `docs/spelling_wordlist.txt` ?

Reply:
yes, you are right here, i have moved these lines to appropriate file